lib/components/topbar/topbar.less

```diff
@@ -424,13 +439,15 @@
         }
     }
 
+    &:has(> &--skip-link:focus) {
+        border-top: none; // Hide the border-top when the skip link is focused
+    }
+
     // STATIC COMPONENT STYLES
     align-items: center;
     background-color: var(--theme-topbar-background-color, var(--white));
     border-bottom: var(--theme-topbar-bottom-border, var(--su-static1) solid var(--black-225));
-    border-top: var(--theme-topbar-accent-border, 3px solid var(--theme-primary));
-    display: flex;
```

Comment:
AFAICT, we don't need to set a `display: flex` on the topbar. If we want to play it safe, we could keep `display: flex` here and override it with `:has` (my new favorite trick!) to change the display in the event the topbar includes `--container` and `--skip-link`.